hopp/simulation/hopp.py

```diff
@@ -15,7 +15,7 @@
 class Hopp(BaseClass):
     name: str = field(converter=str)
     config: dict = field(converter=dict)
-    site: SiteInfo = field()
+    site: SiteInfo = field(converter=SiteInfo.from_dict)
     technologies: dict = field(converter=dict)
 
     def __attrs_post_init__(self) -> None:
```

Comment:
I know this file isn't part of the pull request, but the HybridSimulation has other input configs that aren't being passed in via lines 24-28 (dispatch_options, cost_info, simulation_options)

Reply:
Good call, I will add the ability to pass those inputs as well.